pyproject.toml

```diff
@@ -149,4 +145,7 @@ exclude = ["dcpy.test*"]
 library = "dcpy.library.cli:run"
 
 [tool.setuptools.package-data]
-dcpy = ["library/templates/*.yml"]
+dcpy = [
+    "library/templates/*.yml",
+    "lifecycle/package/resources/data_dictionary_template.jinja",
```

Comment:
is it possible to specify any jinja templates, so we don't continually stub our toe on this?

Reply:
should be! but my first few tries didn't work and I was gonna generalize this in the follow-up work with READMEs. that work will likely add a `readme_template.jinja`